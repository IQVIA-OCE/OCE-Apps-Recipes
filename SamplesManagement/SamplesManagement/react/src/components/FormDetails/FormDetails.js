import React from 'react';
import { View, StyleSheet } from 'react-native';
import { Text, useTheme } from 'apollo-react-native';

const FormDetails = ({ title, children }) => {
  const theme = useTheme();

  return (
    <View style={styles.container}>
      <Text style={[styles.title, { color: theme.colors.tertiary }]}>{title}</Text>
      {children}
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    marginRight: 20,
  },
  title: {
    marginBottom: 5,
  },
});

export default FormDetails;
