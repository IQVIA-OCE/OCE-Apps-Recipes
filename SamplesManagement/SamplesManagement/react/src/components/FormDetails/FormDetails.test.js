import renderer from 'react-test-renderer';
import FormDetails from './FormDetails';
import React from 'react';
import { Text } from 'react-native';

describe('FormDetails', () => {
  it('Should render FormDetails ', () => {
    const tree = renderer
      .create(
        <FormDetails title="title">
          <Text>Text</Text>
        </FormDetails>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
