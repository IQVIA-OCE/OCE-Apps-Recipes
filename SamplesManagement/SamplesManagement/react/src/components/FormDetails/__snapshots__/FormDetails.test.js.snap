// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormDetails Should render FormDetails  1`] = `
<View
  style={
    Object {
      "marginRight": 20,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#4D4D4D",
        "marginBottom": 5,
      }
    }
  >
    title
  </Text>
  <Text>
    Text
  </Text>
</View>
`;
